api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirPersonServiceImpl.java

```diff
@@ -51,4 +52,15 @@ public Person getPersonByUuid(String uuid) {
 		        .map(personTranslator::toFhirResource).collect(Collectors.toList());
 	}
 	
+	@Override
+	protected FhirDao<org.openmrs.Person> getDao() {
+		return fhirPersonDao;
+	}
+	
+	@Override
+	protected OpenmrsFhirTranslator<org.openmrs.Person, Person> getTranslator() {
+		return personTranslator;
+		
+	}
```

Comment:
Instead of adding these getters, please rename the DAO variable to `dao`, the translator variable to `translator` and add a `@Getter(AccessLevel.PROTECTED)` annotation to the class.